modules/social_features/social_out_of_office/config/install/field.field.profile.profile.field_profile_alt_contact.yml

```diff
@@ -0,0 +1,31 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.profile.field_profile_alt_contact
```

Comment:
I think it would be better to prefix all ooo related fields with `ooo`, like `field_profile_ooo_alt_contact`

Please apply to all fields

Reply:
I've applied it to all fields.